x12-edi/2.5/modules/ROOT/Edits/x12-edi-connector-examples.adoc

```diff
@@ -0,0 +1,51 @@
+= X12 EDI Connector 2.5 - Examples - Mule 4
+
+Use this example to view the use case for Business-to-Business (B2B) EDI integration leveraging Anypoint X12 Connector. This example:
+
+* Transforms XML messages into Outbound EDI X12 850 purchase order transactions +
+
+An HTTPS Listener source receives the Enterprise purchase order XML messages from a back-end system. DataWeave and X12 Writes translate the XML message into an X12 v4010 850 Purchase order EDI message.
+
+* Transforms Inbound EDI X12 850 purchase orders into JSON +
+
+An HTTP Listener source receives the raw EDI X12 850 purchase order message.
+X12 Read and DataWeave components pare and translate the EDI data into the JSON format.
+
+* Generates a Functional Acknowledgment (997) +
+
+== Prerequisites
+
+. Open the Example project in Anypoint Studio from Anypoint Exchange. +
+. Install the latest version of Anypoint X12 connector.
+
+== Run and Test the Outbound XML to EDI X12 850
+
+. Run the application in Anypoint Studio.
+
+. Test the Outbound flow by using a REST Client to POST a JSON message to the URL:
+
+`http://localhost:8081/x12/send/enterprise-ob-purchaseorder`
+
+Payload location: `src/main/resources/outbound/Enterprise-Outbound-PO.xml`
```

Comment:
How does this line relate to the narrative above and below it?
Why does it use a colon, and why is it not a complete sentence?